utilities/transactions/transaction_test.cc

```diff
@@ -4559,6 +4566,124 @@ TEST_P(TransactionTest, MemoryLimitTest) {
   delete txn;
 }
 
+// Test WritePreparedTxnDB's IsInSnapshot against different ordering of
+// snapshot, max_committed_seq_, prepared, and commit entries.
+TEST_P(WritePreparedTransactionTest, IsInSnapshotTest) {
```

Comment:
I check the travis job and seems WritePreparedTransactionTest don't include any tests from TransactionTest. In that case you don't even need to have WritePreparedTransactionTest inherit from TransactionTest, and then just hard code the params.